### PR TITLE
Update windows.rst

### PR DESCRIPTION
**Pull request by fialky**
After two days of struggling with the build procedure I include my notes that lead to successful finish.

### CODE REVIEW

File: docs/source/contributing/build/windows.rst [hunk 1]
```diff
@@ -8,19 +8,23 @@ Dependencies
...
 ----------------------------------------------------------
  1. Download precompiled `MediaInfo <https://mediaarea.net/de/MediaInfo/Download/Windows>`_ (DLL)
  2. Download `MediaInfoLib <https://github.com/MediaArea/MediaInfoLib>`_. |br|
     Copy the folder ``Source/MediaInfoDLL`` to ``path/to/MediaElch/MediaInfoDLL``
  3. Download `ZenLib <https://github.com/MediaArea/ZenLib>`_. |br|
-    Copy the folder ``Source/ZenLib`` to ``path/to/MediaElch/ZenLib``
+    Copy the folder ``Source/ZenLib`` to ``path/to/MediaElch/ZenLib`` |br|
+ 4. Download and run `MSYS2 installer <https://www.msys2.org/>`_. |br|
+    Install ``mingw-w64-i686-quazip`` using included `pacman <https://www.msys2.org/>`_ utility |br|
+    Copy the folder ``msys64/mingw32/include/quazip5`` to ``path/to/MediaElch/quazip5`` |br|
```
Comment (bugwelle) [inline docs/source/contributing/build/windows.rst:31]: Interesting. That should not be necessary. MediaElch compiles it's own quazip. Did you also initialize MediaElch's git submodules? :-)

**Review by bugwelle — COMMENTED** [review]
Hi,

thank you for this contribution. I don't work on Windows and never got MediaElch to properly compile there except with this (outdated) guide. I'll try this soon and will merge this PR then.

Thanks!

File: docs/source/contributing/build/windows.rst (hunk 1 above)
Comment (fialky) [reply docs/source/contributing/build/windows.rst:31]: I tried to put contents of quazip-master (https://github.com/stachenov/quazip) into path/to/MediaElch/third_party/quazip. I was also digging around with zlib.h and zconf.h. With this setup it did not compile.

Comment (fialky) [reply docs/source/contributing/build/windows.rst:31]: Forgot to mention zconf.h and zlib.h files for CONFIG += USE_EXTERN_QUAZIP procedure (6e081d2).

Comment (bugwelle) [reply docs/source/contributing/build/windows.rst:31]: Ah, ok. zlib may indeed produce issues...

PS: You can initialize git submodule using `git submodule init`

Comment (fialky) [reply docs/source/contributing/build/windows.rst:31]: Oh, thanks. I did not know `git submodule init`. And after that `git submodule update` cloned quazip into path/to/MediaElch/third_party/quazip. Unfortunately, it did not compile also complaining about undefined references from zlib.h. Copying zlib.h and zconf.h into path/to/MediaElch/third_party/quazip/quazip did not help too.

**Pull request merged**